### ISSUE TITLE
Tuplet enhancements 

### ISSUE THREAD

**Issue by maribote** [Abjad/abjad#1244]
Two enhancement requests:

1. As per https://groups.google.com/g/abjad-user/c/5WCnIAOUK6k when writing `tuplet = abjad.Tuplet((x, y), music)`, `tuplet.denominator` should default to x, following user’s input. The same behavior should be taught to the LilyPondParser for the `\times` syntax. 

2. The LilyPondParser should be taught about the standard `\tuplet` syntax.

**Comment by trevorbaca** [comment]
Yes, these are good and we'll add them in.

LilyPond's `\tuplet` syntax is definitely better than `\times`. The only real question for Abjad implementation is whether to keep the `\times` syntax in Abjad at all.

Will decide after Abjad 3.2.

**Comment by gilbertohasnofb** [comment]
As of LilyPond version 2.20.0, both syntaxes are accepted. Removing `\times` from Abjad would not only break scores unnecessarily, but also remove a valid LilyPond syntax. E.g.:

```
\version "2.20.0"
{
    \times 2/3 {c'2 d'2 e'2}   
    \tuplet 3/2 {f'2 g'2 a'2}
}
```
![image](https://user-images.githubusercontent.com/5700795/101820668-d693cc00-3b1e-11eb-8eae-559e3e116f22.png)

**Comment by trevorbaca** [comment]
Hi @gilbertohasnofb, yes I know this ;)

The question isn't on the input side (parsing, like in your example). The question is on the output.

Right now Abjad outputs only `\times`. When making the tuplet changes I think I'll probably switch this entirely. Then Abjad will output only `\tuplet`.

Make sense?

**Comment by gilbertohasnofb** [comment]
Makes sense Trevor, thanks for the quick reply. I misunderstood the wording of your previous message, didn't catch that you were referring to output only.

**Comment by maribote** [comment]
Closing this issue as both enhancement requests were implemented in abjad 3.2 (thank you!). If abjad should default to \tuplet for output (I also think it should) should be discussed in a separate thread if necessary.
